doc/tutorials/pymunk_platformer/pymunk_demo_platformer_09.py

```diff
@@ -107,9 +107,6 @@ def __init__(self):
         # Set the initial texture
         self.texture = self.idle_texture_pair[0]
 
-        # Hit box will be set based on the first image used.
-        self.hit_box = self.texture.hit_box_points
-
```

Comment:
The hitbox stays stable because if we switch textures and the new hitbox suddenly appears "inside" a solid object the engine doesn't have a good way to resolve the collision. Should it go left? Right? It doesn't know the original direction and how it got there.

Reply:
I think setting the `hit_box` with `self.texture.hit_box_points` throws an error. Because the hit box expects a `RotatableHitBox` and also `Texture` doesn't has a method which returns `RotatableHitBox`.

Reply:
Wait I think `self.set_hit_box` should do it.